.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Integration Tests
```

Comment:
Can you rename this file to `integration.yml`?

Reply:
Done